### PR TITLE
Add diff node module

### PR DESCRIPTION
**Pull request by tbrisker**
For plugin updates, please indicate which repos this should be built into:

* [x] Nightly
* [ ] 1.13
* [ ] 1.12
* [ ] 1.11

See Foreman's [plugin maintainer documentation](http://projects.theforeman.org/projects/foreman/wiki/How_to_Create_a_Plugin#Release-strategies) for more information.

---

### CODE REVIEW

File: nodejs-diff/nodejs-diff.spec [hunk 1]
```diff
@@ -0,0 +1,64 @@
+%global npm_name diff
+%global enable_tests 0
```
Comment (domcleal) [inline nodejs-diff/nodejs-diff.spec]: Should be enabled, it ought to pass.

File: nodejs-diff/nodejs-diff.spec [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+%prep
+%setup -q -n package
+
+%build
+%nodejs_symlink_deps --build
+
+%install
+mkdir -p %{buildroot}%{nodejs_sitelib}/%{npm_name}
+cp -pfr CONTRIBUTING.md LICENSE README.md dist examples images lib package.json release-notes.md runtime.js %{buildroot}%{nodejs_sitelib}/%{npm_name}
+# If any binaries are included, symlink them to bindir here
```
Comment (domcleal) [inline nodejs-diff/nodejs-diff.spec]: Can be removed

**Comment by tbrisker** [comment]
@domcleal Thanks, fixed.

File: nodejs-diff/nodejs-diff.spec [hunk 3]
```diff
@@ -0,0 +1,62 @@
+%global npm_name diff
+%global enable_tests 1
+
+%{?nodejs_find_provides_and_requires}
```
Comment (dLobatog) [inline nodejs-diff/nodejs-diff.spec]: This one can be removed, there's an identical macro in line 22. Other than that :+1:

Comment (tbrisker) [inline nodejs-diff/nodejs-diff.spec]: done

**Comment by domcleal** [comment]
Merged and building, thanks @tbrisker.

**Pull request merged**